python/src/PenalizedLeastSquaresAlgorithm_doc.i.in

```diff
@@ -35,9 +39,16 @@ ApproximationAlgorithm, LeastSquaresMetaModelSelection
 
 Notes
 -----
-Solve the least-squares problem:
+For each output marginal :math:`k \in \{1, ..., n_Y\}`,
+solve the least-squares problem:
 
 .. math::
 
-    \vect{a} = \argmin_{\vect{b} \in \Rset^P} ||y - \vect{b}^{\intercal} \vect{\Psi}(\vect{U})||^2
-"
+    \widehat{\vect{a}} = \argmin_{\vect{a} \in \Rset^P} \left\|\vect{y}_k - \Tr{\mat{\Psi}} \vect{a} \right\|^2
+
+where :math:`\vect{y}_k \in \Rset^{n}` is the :math:`k`-th marginal of the
+sample of output observations,
+:math:`P \in \Nset` is the number of coefficients,
+:math:`\mat{\Psi} \in \Rset^{n \times P}` is the design matrix
+computed from the input sample *x* and
```

Comment:
Is the italic x voluntary because you want to refer to the argument name rather than the mathematical notation?

Reply:
Within the API help page, input and output arguments are formatted like this.
This is for example used in [LeastSquaresExpansion_doc.i.in](https://github.com/openturns/openturns/blob/aeddc45aab96a92bc5210e7f3cb10551f7fe230c/python/src/LeastSquaresExpansion_doc.i.in#L22) for the input sample and the weights.

There are, however, dozens of examples of another formatting, like, for example in [DistributionImplementation_doc.i.in](https://github.com/openturns/openturns/blob/aeddc45aab96a92bc5210e7f3cb10551f7fe230c/python/src/DistributionImplementation_doc.i.in#L197) for the *Xcond* input argument.

It might be worth to ask to the experts: https://github.com/openturns/openturns/issues/2593